client/Carousel.jsx

```diff
@@ -54,86 +67,139 @@ class Carousel extends Component {
   }
 
   getAllFromCategory(event, category) {
-    console.log(category);
     axios.get('/item', { 
       params: {
         Category: category
       }
     })
       .then(data => {
```

Comment:
Consider destructuring here which will allow you to set itemData to data instead of data.data in line 76. Your call!